src/collection.ts

```diff
@@ -35,11 +41,42 @@ export class Collection {
     return `[Collection: ${this.path}]`;
   }
 
+  get subCollections(): Array<Collection> {
```

Comment:
Tiny naming nit, I think this should be `subcollections` instead of `subCollections`. When we write docs I think it reads better as the normal "subcollection" word. Everything else LGTM and I like our broad use of `options` arguments around this codebase.